src/control/SConscript

```diff
@@ -33,6 +33,16 @@ def get_install_bin_path(benv, name):
     """Get the final installation location for a binary with a given name"""
     return join(benv.subst("$PREFIX"), "bin", name)
 
+def is_firmware_mgmt_build(benv):
+    fw_mgmt_enabled = benv.subst("$FIRMWARE_MGMT")
+    print(fw_mgmt_enabled)
```

Comment:
is this just debug or do we want to print

Reply:
Oops, that was indeed debug. Will remove and fix the pylint issues.

Reply:
Fixed.